src/Options/ParseOptions.hpp

```diff
@@ -180,49 +186,55 @@ class Options {
   std::string help() const noexcept;
 
  private:
+  template <typename, typename, typename>
+  friend struct Options_detail::get_impl;
+
   static_assert(tt::is_a<tmpl::list, OptionList>::value,
                 "The OptionList template parameter to Options must be a "
                 "tmpl::list<...>.");
   using opts_t = OptionList;
-  static constexpr int max_label_size_ = 20;
-  static constexpr size_t max_help_size_ = 56;
-
-  void parse(const YAML::Node& node);
+  using subgroups_t = tmpl::remove_duplicates<
+      tmpl::transform<opts_t, Options_detail::find_subgroup<tmpl::_1, Group>>>;
```

Comment:
It looks like this is the only remaining use of `opts_t`, so just use OptionList here and get rid of the old, poorly named alias.